tests/test_wave_06.py

```diff
@@ -129,3 +129,46 @@ def test_swap_best_by_category_no_other_match():
     assert item_a in tai.inventory
     assert item_b in tai.inventory
     assert item_c in tai.inventory
+
+
+
+def test_get_newest():
+    item_a = Clothing(age=2)
+    item_b = Decor(age=14)
+    item_c = Clothing(age=20)
+    item_d = Decor(age=5)
+    item_e = Clothing(age=85)
+    tai = Vendor(
+        inventory=[item_a, item_b, item_c, item_d, item_e]
+    )
+
+    best_item = tai.get_newest()
+
+    assert best_item.age == 2
+
+
+def test_swap_best_by_age():
```

Comment:
Great tests btw.

Suggestion:

1.  Test when each `Vendor` has an empty list of items.
1.  Test when a vendor has multiple items of the smallest age.